### ISSUE TITLE
LM8UU & LME8UU stl's the same!

### ISSUE THREAD

**Issue by indieflow** [josefprusa/PrusaMendel#47]
Hi, just been looking through your files and noticed that the LM8UU & LME8UU pre-generated stl's are the same! Problem is solved by changing false-true in OpenScad, rendering and exporting, though thought I'd point this out for others so they don't print straight from stl's!

**Comment by jglauche** [comment]
I think I found the issue, setting lme8uu=false in the generte lm8uu file will make sure this won't happen again; recompiling the lm8uu files now and will push the changes any minute.

**Comment by jglauche** [comment]
done; I'll verify the files tomorrow.

**Comment by indieflow** [comment]
Thank god for that, as looking into what you requested would of taken me all night! What with been new to this! lol

**Comment by jglauche** [comment]
I would be happy if you could test the re-made files before I can do that tomorrow to verify that they are right.

**Comment by indieflow** [comment]
Right, just had a quick look and seems as though the updated files are right, though you've placed them in the LM8UU section, when they should have gone in the LME8UU section! 

Basically I believe the LM8UU variations are now under the LME8UU section and the LME8UU variations are under the LM8UU section, right files, wrong folders!

**Comment by indieflow** [comment]
I could be getting confused though as I only have the LME8UU linear bearings! Am I right in believing the LME8UU are longer than LM8UU's?

**Comment by jglauche** [comment]
I only updated the lm8uu section. I can confirm that the lm-e-8uu version works. 
So when you have lm-e-8uu's, print the files in the lm-e-8uu folder. The lme8uu are 1mm wider in diameter and 1mm longer than the lm8uu bearings.

**Comment by indieflow** [comment]
I believe it was the LME8UU section that needed updating though, not the LM8UU section! 

I originally noticed the problem when I printed what I believed to be an LME8UU y-bushing from the stl's, though it was too small for my bearings! When I altered the false to true in the scads, the rendered bushing was right. Checking what you've altered I believe you have changed the LM8UU files to LME8UU! 

Also the LME8UU linear bearing slots print out 16mmx26mm when LME8UU bearings measure 16x25mm! Basically giving a slight play length ways, is this intentional or can it be altered?

**Comment by jglauche** [comment]
I built a prusa mendel out of lme8uu parts this weekend with exactly the files in the repository and they all fitted beautifully (using slic3r-0.6.1-dev). Play on y-bushings was 0,2mm. You should try if it can move when you tighten them with zip-ties.

**Comment by indieflow** [comment]
When measuring the stl with Netfabb, I'm getting a bearing cutout space of 16x26mm. Like I say, same when printed! Diameter wise is a perfect fit, though length slightly long! Yes a zip-tie will hold it in position, though just wondered whilst on the topic if the length could be made more accurate.

**Comment by jglauche** [comment]
My prints of lme8uu y-bushings come out with 25,3mm spacing. My guess is that yours come out slightly bigger because of print settings. As long as it works, I'm happy with it.

**Comment by jglauche** [comment]
Okay, I have re-done the STLs again. The problem what happened:
I set lme8uu=true in the configuration.scad on my local copy. The generate-metric-prusa-lm8uu.sh script sets lme8uu=false , this was wokring inside the file to compile (i..e y-bushing.scad) but the bushing.scad loaded from that used lme8uu=true in that case. This may be an openscad bug, however a test case I attempted to make resulted in the expected behaviour. As current workaround I suggest whenever STLs are made, set all variables to false in the configuration.scad file. It seems the issue only occurs when a variable is set to true.
